OmarEspinosaCastro/30-Agosto/ArrayList.java

```diff
@@ -0,0 +1,29 @@
+
+package collection;
+
+public class ArrayList {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        int numberLines = in.nextInt();
+        ArrayList<ArrayList<Integer>> lineas = new ArrayList<>();
```

Comment:
Always using interface type when declaring a collection
https://www.codejava.net/java-core/collections/18-java-collections-and-generics-best-practices